docs/version0.15.md

```diff
@@ -65,10 +66,20 @@ The ChaCha20 and ChaCha20-Poly1305 algorithms can now use OpenSSL on Java 11. Fo
 
 For compatibility, the [`-XX:OnOutOfMemoryError`](xxonoutofmemoryerror.md) OpenJDK Hotspot option is now supported by OpenJ9.
 
-### Support for Transparent HugePage
+###  ![Start of content that applies only to Java 11 (LTS)](cr/java11.png) Support for Transparent HugePage
 
 The VM now supports the allocation of huge pages on Linux when you use the `madvise` (`/sys/kernel/mm/transparent_hugepage/enabled`) setting. To enable this feature, set [`-XX:+TransparentHugePage`](xxtransparenthugepage.md) on the command line when you start your application. This option is currently not enabled by default.
 
+### Support for low-overhead heap profiling
```

Comment:
Not sure if it is intentional that this section doesn't start with `![Start of content that applies only to Java 11 (LTS)](cr/java11.png)`.

In addition, `[Support for Transparent HugePage](#support-for-transparent-hugepage)` has not such decoration while later `Support for Transparent HugePage` has it.

On the other hand, `ChaCha20 algorithm support for OpenSSL` has consistent decoration in two places.

Reply:
Good spot @JasonFengJ9 .... possible badly resolved merge conflict, which release topics suffer from a lot when different people are working on updates. I also spotted that the icon should be an 11+ not just an 11, so hopefully this is now fixed. I'll re-stage so we can check.